packages/melody-compiler/src/state/State.js

```diff
@@ -27,6 +27,10 @@ export default class State {
         this.options = {
             generateKey: true,
             projectRoot: undefined,
+            // eslint-disable-next-line
+            warn: console.warn,
```

Comment:
Not sure when this was fixed but older versions of v8 needed `console.warn.bind(console)`.